Okta/OktaAuth.swift

```diff
@@ -47,6 +47,8 @@ public struct OktaAuthorization {
 
                         // Set the local cache and write to storage
                         OktaAuth.tokens = tokenManager
+                        self.storeAuthState(tokenManager)
```

Comment:
storeAuthState function already has that call - OktaAuth.tokens = tokenManager. Delete line 49?

Reply:
Fixed.